src/pages/VacancyDetailsPage.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Vacancy } from '../types/vacancy';
+import { Applicant } from '../types/applicant';
+import { Box, Stack, Typography } from '@mui/material';
+import { APP_BAR_HEIGHT } from '../constants';
+import CustomizedVacancyTables from '../components/CustomizedVacancyTable';
+import { mockApplicants } from '../mock-data/applicants';
+import { mockVacancies } from '../mock-data/vacancies';
+import { Route } from 'react-router-dom';
+
+export interface VacancyDetailsPageProps {
+  //receivingDate: string; // TODO: Change this later to Date once the backend and database are connected
+  vacancy: Vacancy;
+  //applicants: Applicant[];
```

Comment:
Later on there will be a fetch in this component to get all applicants, so you don´t need them in props.